app/styles/snippet-plugin.scss

```diff
@@ -164,3 +141,62 @@ div[typeof='besluitpublicatie:Documentonderdeel']
     }
   }
 }
+
+.say-snippet-placeholder {
+  display: flex;
+  flex-direction: row;
+  padding: $au-unit-small $au-unit-small * 1.5;
+  background-color: #fff7bb;
```

Comment:
Aren't these colors already a variable? If not talk with saska as it should be a mistake

Reply:
I've checked with Saska but most of these colours aren't in AU.

Reply:
I've heard back now and the intent was to use some different AU colours, so I've changed those now.